pyensign/events.py

```diff
@@ -185,6 +205,108 @@ def mark_subscribed(self, id, ackback_stream):
         self._stream = ackback_stream
         self._state = EventState.SUBSCRIBED
 
+    def __repr__(self):
+        repr = "Event("
+        repr += "data={}, ".format(self.data)
+        repr += "mimetype={}, ".format(self.mimetype)
+        repr += "schema_name={}, ".format(self.type.name)
+        repr += "schema_version={}, ".format(self.type.semver())
+        if self.meta:
+            repr += "meta={}".format(self.meta)
+        repr += ")"
+        return repr
+
+    def __str__(self):
```

Comment:
You can now print events, it will look something like:

```
Event:
	id: b'\x01\x89\xd2\x1a?,A\x03\xf2\x04\xa6yd\xdf\x0b<'
	data: b'message in a bottle'
	mimetype: text/plain
	schema: message v1.0.0
	state: EventState.ACKED
	created: 2023-08-07 17:24:41
	committed: 2023-08-07 17:24:42.930920
```